### PR TITLE
Small improvement to pong example

### PR DESCRIPTION
**Pull request by unbu**
Sets initial y values to middle of game area

### CODE REVIEW

File: examples/04_pong/pong.rs [hunk 1]
```diff
@@ -75,8 +75,8 @@ fn initialise_paddles(world: &mut World) {
     let mut right_transform = LocalTransform::default();
 
     // Correctly position the paddles.
-    left_transform.translation = [0.0, 0.0, 0.0];
-    right_transform.translation = [ARENA_WIDTH - PADDLE_WIDTH, 0.0, 0.0];
+    left_transform.translation = [0.0, ARENA_HEIGHT / 2.0, 0.0];
```
Comment (Rhuagh) [inline examples/04_pong/pong.rs]: This is not consistent. Left paddle will be halfway outside, while right will be wholly inside.

Comment (Rhuagh) [reply examples/04_pong/pong.rs]: And I know that was there before just felt it was a good opportunity to fix it :)

Comment (torkleyy) [reply examples/04_pong/pong.rs]: Are you sure? At least it doesn't seem to be the case right now.

Comment (unbu) [reply examples/04_pong/pong.rs]: Look good to me in-game

Comment (Rhuagh) [reply examples/04_pong/pong.rs]: Well look at the x parameter. One is 0, one is window width - paddle size

Comment (torkleyy) [reply examples/04_pong/pong.rs]: Exactly, which seems right to me.

Comment (Rhuagh) [reply examples/04_pong/pong.rs]: Ah, reference point isn't center point of the paddle?

Comment (unbu) [reply examples/04_pong/pong.rs]: pong

Comment (unbu) [reply examples/04_pong/pong.rs]: How it looks

Comment (Rhuagh) [reply examples/04_pong/pong.rs]: Yeah it's fine, I realized that the paddle isn't centered on its center of mass.

Comment (torkleyy) [reply examples/04_pong/pong.rs]: No, I don't think so. But I think the y coordinate isn't right yet.

Comment (Rhuagh) [reply examples/04_pong/pong.rs]: Yeah, the origin of the paddle is the upper left corner.

Comment (Rhuagh) [reply examples/04_pong/pong.rs]: Or bottom, not sure

Comment (unbu) [reply examples/04_pong/pong.rs]: You are right, They aren't exactly in the center

Comment (unbu) [reply examples/04_pong/pong.rs]: Should be fixed now, hopefully

**Comment by Rhuagh** [comment]
Looks good, thanks!  LGTM!

**Comment by unbu** [comment]
First PR, thanks for pointers.

**Comment by Xaeroxe** [comment]
r? @torkleyy 

@unbu We have developers all around the globe so our policy is to wait 24 hours to make sure everyone's had some waking hours to review the PR.  Once the 24 hours has elapsed I imagine we'll merge this, thanks again!

**Comment by jojolepro** [comment]
:tada: Congrats!

**Pull request merged**